docs/topics/mta-exclude-files-and-packages.adoc

```diff
@@ -0,0 +1,18 @@
+// Module included in the following assemblies:
+//
+// * docs/cli-guide/master.adoc
+
+:_content-type: PROCEDURE
+[id="exclude-files-and-packages_{context}"]
+= Configuring {ProductShortName} to exclude packages and files
+
+
+[id="ignored-locations_{context}"]
+== Searching locations for exclusion
+
+{ProductShortName} searches the following locations:
+
+* `~/.{LC_PSN}/ignore/`
+* `~/.windup/ignore/`
+* `<{ProductShortName}_HOME>/ignore/`
+
```

Comment:
My bad in https://github.com/windup/windup-documentation/pull/848#discussion_r1483582490

I meant that this whole section should be removed

Reply:
Commenting out the section in master.adoc